guide/src/web-sys/overview.md

```diff
@@ -0,0 +1,44 @@
+# `web-sys` Overview
+
+The `web-sys` crate has this file and directory layout:
+
+```text
+.
+├── build.rs
+├── Cargo.toml
+├── README.md
+├── src
+│   └── lib.rs
+└── webidls
+    ├── available
+    │   └── ...
+    └── enabled
+        └── ...
+```
+
+### `webidls/available/*.webidl`
+
+These are all the different WebIDL definitions we intend to support. At the time
+of writing, we don't even attempt to generate bindings for most of these.
+
+### `webidls/enabled/*.webidl`
+
+These are the WebIDL interfaces that we will actually generate bindings for (or
+at least bindings for *some* of the things defined in these files).
+
+These are all symlinks into the `webidls/available` directory.
```

Comment:
Given that windows doesn't support symlinks very well, could we instead perhaps move files from `available` to ` enabled`?